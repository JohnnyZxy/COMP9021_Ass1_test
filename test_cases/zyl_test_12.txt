Suppose you visited the island of Knights and Knaves. 
You come across two of the inhabitants lazily lying in the sun.
You met a group of 6 islanders. 
Their names were Sirs Zoro, Beatrix, Arthur, Michelle, Tracy and Xan. 

Sir Beatrix said:"Sir Zoro is a Knight."
Sir Zoro said:"Sir Arthur is a Knave."
Sir Arthur said: "Sir Beatrix is a Knave."
Sir Tracy said: "Sir Xan is a Knight."
Sir Michelle said: "Sir Tracy is a Knight."
Sir Xan said:" Sir Michelle is a Knight."
Sir Beatrix said: "Sir Xan is a Knave."
Sir Beatrix added:"I am a Knight."