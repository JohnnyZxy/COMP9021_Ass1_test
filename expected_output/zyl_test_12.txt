Which text file do you want to use for the puzzle? The Sirs are: Arthur Beatrix Michelle Tracy Xan Zoro
There are 2 solutions.